integr8ly/installation
# Update kc operator version

## Additional Information
Bumping RHSSO to 1.7.2

## Verification Steps
1. Test Install
2. Check that RHSSO is working as expected. Check the sso metrics endpoint to see if the `request_duration` metric is getting returned (<sso-route>/auth/realms/master/metrics)
2. Test uninstall

## Is an upgrade task required and are there additional steps needed to test this?
Test upgrade from 1.4.1
